test-harness/specs/security-scheme-validation-AC-8.oas3.txt

```diff
@@ -0,0 +1,29 @@
+====test====
+Resource is returned when *X-API-Key* required header is provided.
```

Comment:
We should check for denied as well.

Reply:
added in https://github.com/stoplightio/prism/pull/484/commits/4a0fd4990a575d7a42d26ec28cc5e62dba5c0719#diff-5fe41ed9481abb1f32aa1142c6054827